src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -70,7 +72,12 @@ public CodeGenIL(IDictionary<string, IList> input, string filePath)
             var t = compileResult.tbuilder.CreateType();
```

Comment:
maybe add a new stopwatch that will include `CompileAstToDynamicType` and `mi.Invoke(null, new object[] { null, methodCache, output });` ?
We could have a `CompileAndExecuteTime` separate from the current ExecutionTime